jest.config.js

```diff
@@ -0,0 +1,195 @@
+/* eslint-disable max-len */
+// For a detailed explanation regarding each configuration property, visit:
+// https://jestjs.io/docs/en/configuration.html
+
+module.exports = {
+  // All imported modules in your tests should be mocked automatically
+  // automock: false,
+
+  // Stop running tests after `n` failures
+  // bail: 0,
+
+  // Respect "browser" field in package.json when resolving modules
+  // browser: false,
+
+  // The directory where Jest should store its cached dependency information
+  // cacheDirectory: "/private/var/folders/7f/vrhn6c552bsghd95nn6m7pnc0000gn/T/jest_dx",
+
+  // Automatically clear mock calls and instances between every test
+  clearMocks: true,
+
+  // Indicates whether the coverage information should be collected while executing the test
+  // collectCoverage: false,
+
+  // An array of glob patterns indicating a set of files for which coverage information should be collected
+  collectCoverageFrom: [
+    '**/*.{js,jsx}',
+    '!**/node_modules/**',
+    '!**/tests/**',
+    '!**/public/**',
+    '!**/*.config.js',
+    '!**/database/dbAccessKey.js',
+    '!**/database/imageUrls.js',
+    '!**/*.json',
+  ],
+
+  // The directory where Jest should output its coverage files
+  coverageDirectory: 'tests/coverage',
+
+  // An array of regexp pattern strings used to skip coverage collection
+  // coveragePathIgnorePatterns: [
+  //   "/node_modules/"
```

Comment:
@aku2nnya Do you want it to factor your node modules into your total coverage calculation here-- my only concern is it may sell you short on the true percentage of your code that you have written tests for with how many packages we have!

Reply:
I believe the commented out settings are the default settings that are applied (this was generated by using init). So I should be good here.